README.md

```diff
@@ -1,3 +1,5 @@
+# dex2jar 
+
 **Project move to [SourceForge](https://sourceforge.net/p/dex2jar) and [Bitbucket](https://bitbucket.org/pxb1988/dex2jar) and [Github](https://github.com/pxb1988/dex2jar)**

```

Comment:
Can you also fix the typo here please: `GitHub`

Reply:
Good catch. (: Fixed!